apps/photos/src/components/Sidebar/EnableMap.tsx

```diff
@@ -0,0 +1,40 @@
+import { Stack, Box, Button, Typography } from '@mui/material';
+import Titlebar from 'components/Titlebar';
+import { Trans } from 'react-i18next';
+import { t } from 'i18next';
+
+export default function EnableMap({ onClose, enableMap, onRootClose }) {
+    return (
+        <Stack spacing={'4px'} py={'12px'}>
+            <Titlebar
+                onClose={onClose}
+                title={t('ENABLE_MAPS')}
+                onRootClose={onRootClose}
+            />
+            <Stack py={'20px'} px={'8px'} spacing={'32px'}>
+                <Box px={'8px'}>
+                    {' '}
+                    <Typography color="text.muted">
+                        <Trans i18nKey={'ENABLE_MAP_DESCRIPTION_1'} />
+                        <br />
+                        <br />
+
+                        <Trans i18nKey={'ENABLE_MAP_DESCRIPTION_2'} />
+                        <br />
+                        <br />
+
+                        <Trans i18nKey={'ENABLE_MAP_DESCRIPTION_3'} />
```

Comment:
can we convert this to https://github.com/ente-io/photos-web/blob/6514198cf310b3c9a1c190dd92a8be6efd51a41d/apps/photos/src/components/MachineLearning/MLSearchSettings/enableFaceSearch.tsx#L57-L72